fog/ledger/enclave/api/src/lib.rs

```diff
@@ -32,7 +35,7 @@ pub type Result<T> = StdResult<T, Error>;
 /// gather data that will be encrypted for the client in `outputs_for_client`.
 ///
 /// Key image check is now in ORAM, replacing untrusted
-/// which was doing the check directly.
+/// which was doing the check directly.Sha
```

Comment:
nit: Finish the sentence `Sha...`

Reply:
Addressed in [278d8c1](https://github.com/mobilecoinfoundation/mobilecoin/pull/2896/commits/278d8c1f98696a964ebd09ae87c4b2d5e316cfd0)